v63003/outref/gtm8587.txt

```diff
@@ -0,0 +1,10 @@
+# RUNNING TEST FOR $KEY
+# Check $DEVICE to see it is 0 and $KEY contains the terminating character
+$DEVICE=0
+$KEY=
     (In previous versions, this would be an empty string)
```

Comment:
It looks like an empty string even in the current version. If it is a control character, need a zwrite instead of write so we can see exactly what it has.